src/kbmod/search/KBMOSearch.cpp

```diff
@@ -398,27 +377,8 @@ std::vector<RawImage> KBMOSearch::coaddedScienceStampsGPU(std::vector<Trajectory
     return coaddedScienceStampsGPU(trjs, use_index_vect, params);
 }
 
-// To be deprecated in later PR.
-std::vector<RawImage> KBMOSearch::meanStamps(const std::vector<trajectory>& t_array,
-                                             const std::vector<std::vector<int>>& goodIdx, int radius) {
-    const int num_results = t_array.size();
-    std::vector<TrajectoryResult> arr;
-    for (int s = 0; s < num_results; ++s) {
-        TrajectoryResult trj(t_array[s], goodIdx[s]);
-        arr.push_back(trj);
-    }
-
-    return meanScienceStamps(arr, radius);
-}
-
 RawImage KBMOSearch::summedScienceStamp(const TrajectoryResult& trj, int radius, bool use_all) {
-    return createSummedImage(scienceStamps(trj, radius, false, true, use_all));
-}
-
-// To be deprecated in later PR.
-RawImage KBMOSearch::stackedScience(trajectory& t, int radius) {
-    TrajectoryResult trj(t, stack.imgCount());
-    return createSummedImage(scienceStamps(trj, radius, false, false, true));
+    return createSummedImage(scienceStamps(trj, radius, false, false, use_all));
```

Comment:
```suggestion
    return createSummedImage(scienceStamps(trj, radius, false /*=interpolate*/, false /*=keep_no_data*/, use_all));
```

Reducing to one implementation method looks great, but the calls can be confusing if they're just a few booleans. Adding some context to the method calls would improve readability a lot here.

Reply:
Good point. I expanded on this suggestion by not only including the in-argument comments, but also providing some comments about "why things work they way they do." I spent a lot of time tracing through the functions to understand the historical behavior (which functions used which options), so I think more detail here would be very helpful in the future.